components/Pagination/Pagination.js

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
+import {FormattedMessage, defineMessages, injectIntl, intlShape} from 'react-intl';
 import PropTypes from 'prop-types';
 
+const messages = defineMessages({
+  previousPage: {
+    defaultMessage: "Show Previous Page"
+  },
+  nextPage: {
+    defaultMessage: "Show Next Page"
+  },
+  currentPage: {
+    defaultMessage: "Current Page"
+  }
+});
```

Comment:
Has it been considered to just use icons for that? Like arrows?

Reply:
This is the label for the input field, which is only visible by assistive technology like screen readers.

Reply:
I see, thanks! So, fine from my POV to land, if you are confident that CI didn't break any further.